spec/os.spec.js

```diff
@@ -182,6 +369,28 @@ describe('os.spec: os namespace tests', () => {
             assert.ok(typeof envs == 'object');
             assert.ok(envs['PATH'] || envs['Path']);
         });
+
+        it('contains HOME or USER environment variable', async () => {
```

Comment:
These are POSIX environment variables, so won't work on Windows. Please conditionally check using `NL_OS`. Thanks

Reply:
Hey, using `NL_OS` throws an error so I used `rpm_command` for now as I think that will be common to all OS. None of the Neutralino global variables are being loaded when I print the variables.